core/src/Revolution/modX.php

```diff
@@ -2746,25 +2746,14 @@ protected function _initSession($options = null) {
         $contextKey= $this->context instanceof modContext ? $this->context->get('key') : null;
         if ($this->getOption('session_enabled', $options, true) || isset($_GET['preview'])) {
             if (!in_array($this->getSessionState(), [modX::SESSION_STATE_INITIALIZED, modX::SESSION_STATE_EXTERNAL, modX::SESSION_STATE_UNAVAILABLE], true)) {
-                $sh = false;
-                if ($sessionHandlerClass = $this->getOption('session_handler_class', $options)) {
-                    if ($shClass = $this->loadClass($sessionHandlerClass, '', false, true)) {
-                        if ($sh = new $shClass($this)) {
-                            session_set_save_handler(
```

Comment:
This line is removed but not added back anywhere that I can see - so it seems with this patch we're not overriding the session handler anymore

Reply:
Look at line 2749.

Reply:
I'm not seeing another instance of `session_set_save_handler` anywhere in this diff.

Reply:
Oops, I forgot to move it from my test site. Thanks.

Reply:
Fixed.